### PR TITLE
Make adhesive bandages recipe craft multiples.

### PR DESCRIPTION
**Pull request by TaiGambol**
#### Summary
None

#### Purpose of change
Crafting adhesive bandages from medical gauze causes >95% of the gauze to be wasted. This changes things so that only ~33% is wasted. By contrast, crafting bandages from medical gauze wastes ~22% of the material, so for multiple smaller cuts, ~33% seems reasonable wastage.

#### Describe the solution
Edited the adhesive bandage crafting recipe to craft 30 bandages instead of 1.

#### Describe alternatives you've considered
Changing the recipe to use something other than medical gauze (perhaps a cotton patch? That would make these a field-craftable dressing, now that makeshift bandages are more difficult to craft on the spot)

#### Testing
Created new character with Health Care 2.
Checked crafting screen, found Adhesive Bandage recipe.
Recipe indicates 30 will be created.
Grabbed first aid kit from evac shelter basement.
Crafted adhesive bandages.
30 were crafted.

### CODE REVIEW

**Comment by Kamayana** [comment]
I rather like the idea of being able to make an adhesive bandage with a cotton patch, but that would be in addition to this PR rather than instead of, this making a bunch of them with gauze and tape also makes sense.

**Comment by TaiGambol** [comment]
Not sure what failed, exactly? It looked like something to do with the mutation code, but I didn't touch that...

**Comment by bombasticSlacks** [comment]
You probably need more duct tape/medical tape to make 30 bandages, though otherwise the rationale makes sense.

**Comment by bombasticSlacks** [comment]
Will need to be restarted after #63856

**Comment by TaiGambol** [comment]
How do I restart?

**Comment by bombasticSlacks** [comment]
I'll get it

**Comment by cake-pie** [comment]
Heads up: #63434 changes adhesive bandages from comestible to generic; it will no longer have charges. In case that PR is merged first, change `charges` to `result_mult`.

File: data/json/recipes/other/medical.json [hunk 1]
```diff
@@ -71,10 +71,11 @@
     "skill_used": "firstaid",
     "difficulty": 1,
     "time": "1 m",
+    "charges": 30,
     "batch_time_factors": [ 50, 2 ],
```
Comment (cake-pie) [inline data/json/recipes/other/medical.json:75]: Should probably adjust time and batch time factors as well

File: data/json/recipes/other/medical.json [hunk 2]
```diff
@@ -71,10 +71,11 @@
     "skill_used": "firstaid",
     "difficulty": 1,
     "time": "1 m",
```
Comment (bombasticSlacks) [inline data/json/recipes/other/medical.json:73]:
```suggestion
    "time": "20 m",
```

File: data/json/recipes/other/medical.json (hunk 1 above)
Comment (bombasticSlacks) [inline data/json/recipes/other/medical.json:75]:
```suggestion
```

**Review by bombasticSlacks — COMMENTED** [review]
Going to increase time and merge it so that this can get in

File: data/json/recipes/other/medical.json [hunk 3]
```diff
@@ -70,11 +70,11 @@
     "subcategory": "CSC_*_NESTED",
     "skill_used": "firstaid",
     "difficulty": 1,
-    "time": "1 m",
-    "batch_time_factors": [ 50, 2 ],
+    "time": "20 m",
+    "charges": 30,
```
Comment (bombasticSlacks) [inline data/json/recipes/other/medical.json:74]:
```suggestion
    "result_mult": 30,
```

**Pull request merged**